src/Board/Boards.js

```diff
@@ -13,51 +13,75 @@ import Header from "./Header";
 
 const Boards = () => {
   const dispatch = useDispatch();
-  const [toggleDelete, setToggleDelete] = useState(false);
+  const [visibleDelete, setVisibleDelete] = useState(false);
   const [param, setParam] = useState(false);
-  const stateList = useSelector((state) => state.lists);
+  // const [hidden, setHidden] = useState([]);
+  let [idDisabled, setIdDisabled] = useState();
   const stateBoard = useSelector((state) => state.boards);
+  const stateList = useSelector((state) => state.lists);
   const stateTask = useSelector((state) => state.tasks);
+  const classNames = require("classnames");
+  const [isFetchingBoards, setFetchingBoards] = useState(false);
+
+  useEffect(() => {
+    async function fetchData() {
+      setFetchingBoards(true);
```

Comment:
We need to move loading state (true/false) into store into corresponding reducer.

This comment applies for all other places where loading state is local in the component.